src/discounts/views/SaleCreate/SaleCreate.tsx

```diff
@@ -117,6 +118,16 @@ export const SaleCreateView: React.FC<SaleCreateProps> = ({ params }) => {
           toggleAll={toggleAllChannels}
         />
       )}
+      <DiscountRules
+        rules={[
+          {
+            id: "1",
+            name: "Rule 1",
+            description: "Rule 1 description",
+          },
+        ]}
+        onRuleAdd={() => {}}
+      />
```

Comment:
**issue (blocking)**: we can't merge to main a component which is WIP. This should be removed or hidden behind feature flag.
<img width="1713" alt="image" src="https://github.com/saleor/saleor-dashboard/assets/41952692/6f972d50-e659-4831-a91c-3186a15901f5">